README.md

```diff
@@ -56,10 +56,10 @@ which is either taken from `(atlas.query.validate).value_type` proto field optio
 plugin based on the field type if the option is not supplied. Currently *value_type* can be either STRING or NUMBER.
 The following table shows what is allowed for each *value_type*:
 
-|                                     | STRING | NUMBER |
-|-------------------------------------|--------|--------|
-| **Filtering operators**                 | EQ, MATCH, GT, GE, LT, LE | EQ, GT, GE, LT, LE |
-| **Filtering value type/condition type** | String, null/StringCondition, NullCondition | Number, null/NumberCondition, NullCondition       |
+|                                     | STRING | NUMBER | ARRAY|
+|-------------------------------------|--------|--------|------|
+| **Filtering operators**                 | EQ, MATCH, GT, GE, LT, LE, IEQ | EQ, GT, GE, LT, LE | IN |
+| **Filtering value type/condition type** | String, null/StringCondition, NullCondition | Number, null/NumberCondition, NullCondition       | Number or String array
 
```

Comment:
They way I understand `value type` <-> `proto definition`:
STRING  <->  string name = 1;
NUMBER <->  int age = 2;
ARRAY <->  repeated string nicknames = 3;

but we still want filter like `name in ["John", "Alex"]`

Reply:
In this table  examples are not exist, i think we can add type  repeated before Number and String, we have more clear examples in atlas-app-toolkit in filtering section